frontends/web/src/routes/account/account.tsx

```diff
@@ -84,12 +86,14 @@ class Account extends Component<Props, State> {
         exported: '',
         accountInfo: undefined,
         fatalError: false,
+        deviceIDs: Object.keys(this.props.devices),
```

Comment:
this probably doesn't work when the devices prop is changed.

you can just add a `private deviceIDs = () => { return Object.keys(this.props.devices); }` function and call it whenever needed.